flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/OperatorChain.java

```diff
@@ -549,7 +549,7 @@ public void collect(StreamRecord<T> record) {
 		@Override
 		public <X> void collect(OutputTag<X> outputTag, StreamRecord<X> record) {
 			if (this.outputTag == null || !this.outputTag.equals(outputTag)) {
-				// we are only responsible for emitting to the side-output specified by our
+				// we are not responsible for emitting to the side-output specified by this
```

Comment:
I don't think the origin comment is wrong. Same to `RecordWriterOutput.java`

Reply:
I think if the output tag is null, this output should be responsible for emitting to the main output instead of one side output.

Reply:
The original comment was correct, but I like the new wording and think it is more helpful than the previous one.